modules/gapi/perf/cpu/gapi_video_perf_tests_cpu.cpp

```diff
@@ -105,11 +105,10 @@ INSTANTIATE_TEST_CASE_MACRO_P(WITH_VIDEO(BackgroundSubtractorPerfTestCPU),
                               BackgroundSubtractorPerfTest,
                               Combine(Values(cv::gapi::video::TYPE_BS_MOG2,
                                              cv::gapi::video::TYPE_BS_KNN),
+                                      Values("cv/video/768x576.avi", "cv/video/1920x1080.avi"),
                                       testing::Bool(),
                                       Values(0., 0.5, 1.),
-                                      Values("cv/video/768x576.avi", "cv/video/1920x1080.avi"),
-                                      Values(1, 10),
+                                      Values(5),
```

Comment:
Variable can be initialized in test body. Or, will test be expanded?

Reply:
It can be, by someone who need to measure perf on different amounts of frames. I think we should leave it as a parameter

Reply:
Ok.